Assignment/solution_student/index.html

```diff
@@ -1,35 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
 
-Sample Content
-——-
+  <head>
 
-ABOUT ME PAGE
+    <meta charset="UTF-8">
+    <title>Joe Johnson</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,600italic,700,700italic,800,800italic|Lora:400,400italic,700,700italic' rel='stylesheet' type='text/css'>
+  
+  </head>
+  
+  <header>
 
-I ENJOY LIFE AS A DEVELOPER
+    <div class="border"></div>
 
-I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.
+    <nav>
 
+      <a href="index.html">About Me</a> / <a href="portfolio.html">Portfolio</a>
+      <h1>Joe Johnson</h1>
 
-RESUME PAGE
+      <hr>
 
-Experience
-        
-I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.
-    
-Relevant Experience
-    
-Web Developer, Relaxr 2015 - present
-Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.
+    </nav>
 
-Web Developer, Startup Matchmaker 2015 - present
-Used a design team's wireframes to develop this company's responsive homepage.
+  </header>
 
-Web Developer, CitiPix 2015 – present
-Built web app prototype that allows users to store and quickly retrieve photos using keywords.
+  <body>
```

Comment:
Ooops! The `<body>` tag contains **all** of the content of the site, remember? So the opening `<body>` should be the very first thing you write after the closing `</head>` — you might have gotten the `<header>` mixed up with the `<head>`, but just remember the two _are_ different.